common/views/components/IIIFImagePreview/IIIFImagePreview.js

```diff
@@ -1,52 +1,69 @@
 // @flow
-import {
-  iiifImageTemplate,
-  convertImageUri,
-  convertIiifUriToInfoUri,
-} from '../../../utils/convert-image-uri';
-import ImageViewer from '../ImageViewer/ImageViewer';
-import { classNames } from '../../../utils/classnames';
-import { imageSizes } from '../../../utils/image-sizes';
+import { iiifImageTemplate } from '../../../utils/convert-image-uri';
+import { trackEvent } from '../../../utils/ga';
+import styled from 'styled-components';
+import NextLink from 'next/link';
+import Image from '../Image/Image';
+import Control from '../Buttons/Control/Control';
 
 type Props = {|
   id: string,
   title: string,
   iiifUrl: string,
   width?: number,
+  itemUrl: any,
```

Comment:
What's the reason for this not being `string`?

Reply:
it's a NextLinkType, have added